react-datetime.d.ts

```diff
@@ -77,27 +77,27 @@ declare module ReactDatetime {
      Define the dates that can be selected. The function receives (currentDate, selectedDate)
      and should return a true or false whether the currentDate is valid or not. See selectable dates.
      */
-    isValidDate?: (x: string) => void;
+    isValidDate?: (x: any, y: any) => boolean;
```

Comment:
Are `x` and `y` very good names for the parameters? In this case we know that `x` represents the `currentDate`, shouldn't we use that as the name of the parameter then instead? This should apply to all definitions in this file then.